block_additional_mappings/smpte-st12-1-timecode.md

```diff
@@ -21,9 +21,46 @@ For convenience, here are the time address bit assignments as described in [@?RF
 | 56--57        | Tens of hours          |
 Table: SMPTE ST 12-1 Time Address Bit Positions{#ST12Bits}
 
-For example, a timecode value of "07:32:54;18" can be expressed as a 64-bit SMPTE 12M value as:
+For example, a timecode value of "07:12:26;18" can be expressed as a 64-bit SMPTE 12M value as:
 
 ```
 10000000 01100000 01100000 01010000
 00100000 00110000 01110000 00000000
 ```
+
+Or with the irrelevant bits marked with an "x" which gives 26 usable bits:
+
+```
+1000xxxx 01xxxxxx 0110xxxx 010xxxxx
+0010xxxx 001xxxxx 0111xxxx 00xxxxxx
+```
+
+This is interpreted in hexadecimal:
+
+- 0x8 units of frames
+- 0x1 tens of frames
+- 0x6 units of seconds
+- 0x2 tens of seconds
+- 0x2 units of minutes
+- 0x1 tens of minutes
+- 0x7 units of hours
+- 0x0 tens of hours
+
+Given no value is above 9, the BCD coding correspond to the actual values:
+
+- 8 units of frames
+- 1 tens of frames
+- 6 units of seconds
+- 2 tens of seconds
+- 2 units of minutes
+- 1 tens of minutes
+- 7 units of hours
+- 0 tens of hours
+
+Or
```

Comment:
Nit: `Or` > `Or:` as for the two previous lists.

Reply:
Indeed, that's more coherent.